vault/diagnose/diagnose_tls_test.go

```diff
@@ -0,0 +1,146 @@
+package diagnose
+
+import (
+	"testing"
+
+	"github.com/hashicorp/vault/command/server"
+	"github.com/hashicorp/vault/internalshared/configutil"
+	"github.com/hashicorp/vault/internalshared/listenerutil"
+	"github.com/hashicorp/vault/vault"
+)
+
+func setup(t *testing.T) *vault.Core {
+	serverConf := &server.Config{
```

Comment:
it doesn't look like this logs or errors, so we don't need `t.Helper()`?

Reply:
Yep! That was my understanding as well.